Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+
+    var description: String = ""
+    var timestamp: String = ""
+
+}
+
+struct AccountBalance {
+
+    var brbs: String = ""
+    var cityBucks: String = ""
+    var laundry: String = ""
+    var swipes: String = "0"
+
+}
+
+enum Stages {
+
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+
+}
+
+//MARK: -
+//MARK: Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
+    static func saveToKeychain(loginInfo: LoginInfo) {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = loginInfo.netid as AnyObject
+        keychain["password"] = loginInfo.password as AnyObject
+    }
+    
+    static func removeKeychainLoginInfo() {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = nil
+        keychain["password"] = nil
+    }
+    
+    static func loadFromKeychain() -> LoginInfo? {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        guard let netid = keychain["netid"] as? String, let password = keychain["password"] as? String else {
+            return nil
+        }
+        return (netid: netid, password: password)
+    }
+    
+}
+
+private let loginURLString = "https://get.cbord.com/cornell/full/login.php?mobileapp=1"
```

Comment:
Perhaps convert this to a `URL` instead of a string that we later parse into a URL?